test/test_smoke.py

```diff
@@ -179,6 +189,18 @@ def test_image_boots(image_type):
         exit_status, output = test_vm.run("echo hello", user=image_type.username, password=image_type.password)
         assert exit_status == 0
         assert "hello" in output
+        exit_status, output = test_vm.run("df --output=target,size", user=image_type.username, password=image_type.password)
+        assert exit_status == 0
+        assert "/var/log" in output
+
+        # Figure out how big / is and make sure it is > 10GiB
+        # Note that df output is in 1k blocks, not bytes
+        for line in output.splitlines():
+            fields = line.split()
+            if fields[0] == "/sysroot":
+                size = int(fields[1])
+                assert size > 10 * 1024 * 1024
+                break
```

Comment:
`lsblk -J` might be more useful and manageable here but I don't mind keeping it like this.

Reply:
yeah, at this point I don't see any reason to have to parse json. Maybe if more things needs checking that will make sense. The --output option is df is really nice to keep things consistent.